utils/core/src/serde/mod.rs

```diff
@@ -113,17 +197,16 @@ impl<T: Serializable> Serializable for &Vec<T> {
     }
 }
 
-impl<T: Serializable, const N: usize> Serializable for Vec<[T; N]> {
```

Comment:
I added a blanket implementation for `impl<T, C> Serializable for [T; C]`. Together with the existing blanked implementation for `Vec<T>`, `&Vec<T>`, and `&[T]`, it made these 3 implementations obsolete.